public/orbs-public-blockchain.md

```diff
@@ -149,52 +147,60 @@ The content of the `orbs-node.json` should be:
-* `<ssh access public key file>` - The SSH public key filename (if omitted, defaults to `~/.ssh/id_rsa.pub`)
-* `<orbs node ethereum address>` - The Orbs Node Ethereum address, EIP-55 compliant with checksum capitalization __but without the leading '0x'__
-* `<aws region>` - An AWS region name. The new node will be provisioned in this region. e.g. `ca-central-1`
-* `<node ip>` - A static IP address where this Orbs Node will be reachable. Must be a valid Elastic IP allocated and unattached in the selected region.
-* `<ethereum endpoint url>` - a URL to a working and synced Ethereum node. 
-* `<ssh source cird block>` - One or more CIDR blocks which will be granted access to ssh from into the node. You will still need the public key to connect - it is required only in cases of troubleshooting. The format is standard CIDR. Any IP not in the range will not be able to SSH to the node, even if it has the SSH key file. To allow ssh access from any ip use `"incomingSshCidrBlocks": ["0.0.0.0/0"],` 
-
-### Deploy the Node using Polygon CLI 
-
-Temporarily store the Orbs Node private key in a file `<orbs private key file>` __without the leading 0x__.
+
+- `<orbs node name>` - Will be used with some AWS resource names as prefix or suffix to uniquely identify node resources. Must be S3 bucket name complient (e.g. no \_...)
+- `<aws profile>` - use "default", or, if you are using AWS profiles with more than one AWS account, an AWS profile name with pre-configured AWS credentials and account for your account.
+- `<ssh access public key file>` - The SSH public key filename (if omitted, defaults to `~/.ssh/id_rsa.pub`)
+- `<orbs node ethereum address>` - The Orbs Node Ethereum address, EIP-55 compliant with checksum capitalization **but without the leading '0x'**
+- `<aws region>` - An AWS region name. The new node will be provisioned in this region. e.g. `ca-central-1`. Ensure it is the same region as your previously created Elastic IP address.
+- `<node ip>` - A static IP address where this Orbs Node will be reachable. Must be a valid Elastic IP allocated and unattached in the selected region.
+- `<ethereum endpoint url>` - a URL to a working and synced Ethereum node (such as an Infura endpoint).
+- `<ssh source cird block>` - One or more CIDR blocks which will be granted access to ssh from into the node. You will still need the public key to connect - it is required only in cases of troubleshooting. The format is standard CIDR. Any IP not in the range will not be able to SSH to the node, even if it has the SSH key file. To allow ssh access from any ip use `"incomingSshCidrBlocks": ["0.0.0.0/0"],`
+
+### Deploy the Node using Polygon CLI
+
+Temporarily store the Orbs Node private key in an environment variable **without the leading 0x**. This is to prevent your private key being inadvertently commited to source control.
```

Comment:
Changed here as best practice IMO is to not create the file in the first place as always at risk of inadvertently commiting by mistake. Use local env var instead